clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/caching/providers/TitusClusterProvider.groovy

```diff
@@ -184,6 +190,20 @@ class TitusClusterProvider implements ClusterProvider<TitusCluster>, ServerGroup
     serverGroup.placement.region = region
     serverGroup.scalingPolicies = serverGroupData.attributes.scalingPolicies
     serverGroup.targetGroups = serverGroupData.attributes.targetGroups
+
+    getTaggedImageAttributes(account, job).ifPresentOrElse(
+      { it ->
+        serverGroup.buildInfo.jenkins =
+          [
+            "name"  : it.jenkinsJob ?: "NA",
```

Comment:
Curious for the "NA" value as if the attributes aren't there weren't going to return an empty map.

Why not just leave them null?

Reply:
yeah makes sense , i'll leave them null.